components/competitors/PopularUseCase.vue

```diff
@@ -40,6 +41,10 @@ export default {
       type: Array,
       required: true,
     },
+    numberOfCol: {
+      type: Number,
+      default: 4,
```

Comment:
ideally default should not be `4`

Reply:
It should be 4. It's actually the bootstrap col no. 
I will rename it so, that it's not confusing.